src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
```

Comment:
**quantity** is not the inner Fruit property. Remove this field. Storage responsible for quantity